CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+### v0.35.1
+###### CloudFormation Specifications
+- Update CloudFormation specs to 18.1.0 (pull #[1671](https://github.com/aws-cloudformation/cfn-python-lint/pull/1671))
+- Expand `AllowedValues` for `AWS::Glue::Connection.ConnectionInput.ConnectionType` (pull #[1673](https://github.com/aws-cloudformation/cfn-python-lint/pull/1673))
+- Expand `AllowedValues` for `AWS::Config::ConfigurationRecorder.ResourceTypes` (pull #[1664](https://github.com/aws-cloudformation/cfn-python-lint/pull/1664))
+- Expand `AllowedValues` for `AWS::CloudFront::Distribution.MinimumProtocolVersion` (pull #[1661](https://github.com/aws-cloudformation/cfn-python-lint/pull/1661))
```

Comment:
wouldn't mind combining these into a single line with how many we have per release